run.sh

```diff
@@ -233,9 +233,9 @@ fi
 # fix for broken steamcmd app_info_print: execute install/update manually, checking for updates fails.
 # https://github.com/ValveSoftware/steam-for-linux/issues/9683#issuecomment-1826928761
 if [ ! -f "$ARKSERVER/steamapps/appmanifest_376030.acf" ]; then
-  arkmanager install
+  arkmanager install --verbose
```

Comment:
what about making this optional with a variable defaulting to `""` to keep the current behaviour.